src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+var setval = require('setval')
```

Comment:
As I said, setval as it currently is needs way more documentation and especially readable variable names. To be honest, I have no idea what it actually does and every developer should either be aware of what happens in the codebase or is able to learn it quickly.